kvbc/include/ClientImp.h

```diff
@@ -40,7 +40,9 @@ class ClientImp : public IClient {
 
  protected:
   ClientImp() = default;
-  ~ClientImp() override = default;
+  ~ClientImp() override {
+    if (comm_) delete comm_;
```

Comment:
I suggest to add a task to memory mangement to transform comm_ to smart pointers.

Reply:
yeah, I suggested it to Eyal, it will be done according to priority